### PR TITLE
Update Esperanto (eo_XX) translation

### PR DESCRIPTION
**Pull request by RobinvanderVliet**

### CODE REVIEW

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -4532,51 +4548,51 @@ msgstr ""
...
 
 msgctxt "feedback form"
 msgid "Want to tell us a bit more?"
-msgstr ""
+msgstr "Ĉu vi volas rakonti iom pli?"
 
 #. e.g. Video search viewing options: http://i.imgur.com/ofZeLRn.png
 msgid "Watch Here"
-msgstr "Spektu Ĉi Tie"
+msgstr "Spektu ĉi tie"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4579]:
```suggestion
msgstr "Spektu Ĉi Tie"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -4532,51 +4548,51 @@ msgstr ""
...
-msgstr ""
+msgstr "Spekti sur %s"
 
 msgid "Watch on YouTube"
 msgstr "Spektu ĉe YouTube"
 
 msgctxt "SERP footer content"
 msgid "We Protect Your Privacy"
-msgstr ""
+msgstr "Ni protektas vian privatecon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4589]:
```suggestion
msgstr "Ni Protektas Vian Privatecon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -4726,11 +4744,11 @@ msgstr "Retaj ligiloj"
 
 msgctxt "maps_places"
 msgid "Website"
-msgstr ""
+msgstr "Retejo"
 
 msgctxt "settings"
 msgid "Welcome Message"
-msgstr ""
+msgstr "Bonveniga mesaĝo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4751]:
```suggestion
msgstr "Bonveniga Mesaĝo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -4742,7 +4760,7 @@ msgstr "Bonvenon al la Anasflanko!"
 
 msgctxt "maps_places"
 msgid "What People Say"
-msgstr ""
+msgstr "Kion homoj diras"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4763]:
```suggestion
msgstr "Kion Homoj Diras"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -4793,17 +4811,17 @@ msgstr "Kiun eksteran liveranton oni uzu por vojpriskriboj"
...
-msgstr "Blanke"
+msgstr "Blanka"
 
 msgctxt "color"
 msgid "White"
 msgstr "Blanka"
 
 msgid "Who We Are"
-msgstr ""
+msgstr "Kiuj ni estas"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4821]:
```suggestion
msgstr "Kiuj Ni Estas"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -4793,17 +4811,17 @@ msgstr "Kiun eksteran liveranton oni uzu por vojpriskriboj"
...
 
 msgid "Why Privacy"
-msgstr ""
+msgstr "Kial privateco"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:4824]:
```suggestion
msgstr "Kial Privateco"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -1345,14 +1353,14 @@ msgstr "Agordoj de DuckDuckGo"
 
 msgctxt "forecast"
 msgid "E"
-msgstr ""
+msgstr "E"
 
 msgctxt "settings"
 msgid "EU Android Welcome Message"
-msgstr ""
+msgstr "EU Android Bonveniga mesaĝo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1360]:
```suggestion
msgstr "EU Android Bonveniga Mesaĝo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -1363,17 +1371,17 @@ msgstr ""
...
 #. Form field for your email ;)
-#. From <a href="https://duckduckgo.com/newbang.html ">new !bang page</a> and <a href="https://duckduckgo.com/feedback.html">feedback page</a>. 
+#. From <a href="https://duckduckgo.com/newbang.html ">new !bang page</a> and <a href="https://duckduckgo.com/feedback.html">feedback page</a>.
 msgid "Email"
 msgstr "Retpoŝto"
 
 msgctxt "precise_user_location"
 msgid "Enable Location"
-msgstr ""
+msgstr "Ŝalti lokon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1380]:
```suggestion
msgstr "Ŝalti Lokon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 9]
```diff
@@ -1363,17 +1371,17 @@ msgstr ""
...
 
 msgctxt "new tab page"
 msgid "Enable Most Visited Sites"
-msgstr ""
+msgstr "Ŝalti Plej ofte vizititaj retejoj"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1384]:
```suggestion
msgstr "Ŝalti Plej Ofte Vizititaj Retejoj"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 10]
```diff
@@ -1391,15 +1399,15 @@ msgstr "Ĉifri konektojn"
 #. shown after the DuckDuckGo extension is installed
 msgctxt "welcome message"
 msgid "Encrypts connections when possible"
-msgstr ""
+msgstr "Ĉifras konektojn kiam eblas"
 
 msgid "Enjoy Peace of Mind"
-msgstr ""
+msgstr "Ĝuu mensan pacon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1405]:
```suggestion
msgstr "Ĝuu Mensan Pacon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 11]
```diff
@@ -1442,7 +1450,7 @@ msgstr "Distriĝo"
 
 msgctxt "SERP footer content"
 msgid "Escape The Filter Bubble"
-msgstr ""
+msgstr "Eskapu la filtrilan bobelon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1453]:
```suggestion
msgstr "Eskapu La Filtrilan Bobelon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 12]
```diff
@@ -1460,14 +1468,14 @@ msgstr ""
 
 msgctxt "mobile expanded map"
 msgid "Expand Map"
-msgstr ""
+msgstr "Etendi la mapon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1471]:
```suggestion
msgstr "Etendi La Mapon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 13]
```diff
@@ -1482,18 +1490,18 @@ msgstr "Prisondo"
 
 msgctxt "feedback form"
 msgid "Feedback Sent"
-msgstr "Prijuĝa komento sendita"
+msgstr "Prikomentado sendita"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1493]:
```suggestion
msgstr "Prikomentado Sendita"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 14]
```diff
@@ -1482,18 +1490,18 @@ msgstr "Prisondo"
...
 
 msgctxt "feedback form"
 msgid "Feedback Sent Successfully"
-msgstr ""
+msgstr "Komento sukcese sendita"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1497]:
```suggestion
msgstr "Komento Sukcese Sendita"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 15]
```diff
@@ -1605,55 +1613,57 @@ msgstr "Partoprenu"
 
 msgctxt "SERP footer content"
 msgid "Get New Themes"
-msgstr ""
+msgstr "Ekhavi novajn etosojn"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1616]:
```suggestion
msgstr "Ekhavi Novajn Etosojn"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 16]
```diff
@@ -1605,55 +1613,57 @@ msgstr "Partoprenu"
...
 
 msgctxt "showcase_app"
 msgid "Get Our App & Extension"
-msgstr "Ricevi niajn aplikaĵon & kromprogramon"
+msgstr "Ricevu niajn aplikaĵon kaj kromprogramon"
 
 msgctxt "showcase_privacy"
 msgid "Get Privacy Tips"
-msgstr " Ricevi privatecajn konsiletojn"
+msgstr "Ricevi privatecajn konsiletojn"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1624]:
```suggestion
msgstr "Ricevi Privatecajn Konsiletojn"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 17]
```diff
@@ -1605,55 +1613,57 @@ msgstr "Partoprenu"
...
 msgid "Get your friends to switch and help us grow!"
 msgstr "Konvinku viajn amikojn ekuzi DuckDuckGo kaj helpu nin grandiĝi!"
 
 msgid "Go to Options."
-msgstr "Iru al Opcioj"
+msgstr "Iri al Opcioj."
 
 msgid "Go to Search Engine."
-msgstr "Iri al la serĉilo"
+msgstr "Iri al la serĉilo."
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1666]:
```suggestion
msgstr "Iri al la Serĉilo."
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 18]
```diff
@@ -1719,26 +1729,26 @@ msgstr "Helpo"
...
 
 msgctxt "feedback form"
 msgid "Help DuckDuckGo%s improve searches like this"
-msgstr ""
+msgstr "Helpu DuckDuckGo%s plibonigi tiajn serĉojn"
 
 msgctxt "settings"
 msgid "Help Improve DuckDuckGo"
-msgstr "Helpu Plibonigi je DuckDuckGo"
+msgstr "Helpi plibonigi DuckDuckGo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1740]:
```suggestion
msgstr "Helpi Plibonigi DuckDuckGo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 19]
```diff
@@ -1719,26 +1729,26 @@ msgstr "Helpo"
...
 
 msgctxt "SERP footer content"
 msgid "Help Spread DuckDuckGo"
 msgstr "Helpu disvastigi DuckDuckGo"
 
 msgid "Help Spread Privacy"
-msgstr ""
+msgstr "Helpu disvastigi privatecon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1747]:
```suggestion
msgstr "Helpu Disvastigi Privatecon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 20]
```diff
@@ -1751,13 +1761,13 @@ msgstr "Jen kelkaj aferoj kiujn ni faris, kiuj povus plaĉi al vi."
 #. shown after the DuckDuckGo extension is installed
 msgctxt "welcome message"
 msgid "Here's what it does:"
-msgstr ""
+msgstr "Jen kion ĝi faras:"
 
 msgctxt "new tab page"
 msgid "Hide Most Visited Sites"
-msgstr ""
+msgstr "Kaŝi plej vizititajn lokojn"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1768]:
```suggestion
msgstr "Kaŝi Plej Vizititajn Lokojn"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 21]
```diff
@@ -1768,7 +1778,7 @@ msgstr "Kaŝi serĉvortojn de la retumila titolo kaj de la historio"
 
 msgctxt "video-resolution"
 msgid "High Definition"
-msgstr "alta distingivo"
+msgstr "Alta distingivo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1781]:
```suggestion
msgstr "Alta Distingivo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 22]
```diff
@@ -1790,36 +1800,36 @@ msgstr "Honkongo"
...
 
 msgctxt "Place detail"
 msgid "Hours may not reflect changes made due to"
-msgstr ""
+msgstr "Horoj eble ne reflektas ŝanĝojn faritajn pro"
 
 msgctxt "SERP footer content"
 msgid "How We Are Profitable"
-msgstr ""
+msgstr "Kiel ni estas profitdonaj"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1811]:
```suggestion
msgstr "Kiel Ni Estas Profitdonaj"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 23]
```diff
@@ -1830,28 +1840,28 @@ msgid "How is it anonymous?"
 msgstr "Kiel estas sennome?"
 
 msgid "How it Works"
-msgstr ""
+msgstr "Kiel ĝi funkcias"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1843]:
```suggestion
msgstr "Kiel ĝi Funkcias"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 24]
```diff
@@ -1903,11 +1912,11 @@ msgstr ""
 
 msgctxt "settingsvalue"
 msgid "Image Tiles"
-msgstr ""
+msgstr "Bildaj tegoloj"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:1915]:
```suggestion
msgstr "Bildaj Tegoloj"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 25]
```diff
@@ -2170,11 +2179,11 @@ msgstr "Pli da %sinformo%s"
 
 msgctxt "SERP footer content"
 msgid "Learn About Bangs"
-msgstr "Lerni Pri Krisignoj"
+msgstr "Ekscii pli pri krisignoj"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:2182]:
```suggestion
msgstr "Ekscii Pli Pri Krisignoj"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 26]
```diff
@@ -2170,11 +2179,11 @@ msgstr "Pli da %sinformo%s"
...
 
 msgctxt "SERP footer content"
 msgid "Learn About DuckDuckGo"
-msgstr ""
+msgstr "Eksciu pli pri DuckDuckGo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:2186]:
```suggestion
msgstr "Eksciu Pli Pri DuckDuckGo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 27]
```diff
@@ -2309,24 +2318,24 @@ msgstr "Ĉu vi amas DuckDuckGo?"
...
+msgstr "Malajzio (angla)"
 
 #. Used in 0-click box for local results, e.g. <a href="http://duckduckgo.com/?q=black+lab+bistro">http://duckduckgo.com/?q=black+lab+bistro</a>
 msgid "Map"
 msgstr "Mapo"
 
 msgctxt "settings"
 msgid "Map Rendering"
-msgstr ""
+msgstr "Mapa bildigo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:2338]:
```suggestion
msgstr "Mapa Bildigo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 28]
```diff
@@ -2862,19 +2871,19 @@ msgstr ""
 
 msgctxt "precise_user_location"
 msgid "Other."
-msgstr ""
+msgstr "Alia."
 
 msgctxt "SERP footer content"
 msgid "Our Crash Course"
-msgstr ""
+msgstr "Nia rapidkurso"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:2878]:
```suggestion
msgstr "Nia Rapidkurso"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 29]
```diff
@@ -3091,7 +3100,7 @@ msgid "Privacy"
 msgstr "Privateco"
 
 msgid "Privacy Blog"
-msgstr ""
+msgstr "Blogo pri privateco"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3103]:
```suggestion
msgstr "Blogo Pri Privateco"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 30]
```diff
@@ -3100,10 +3109,10 @@ msgid "Privacy Browser Extension"
 msgstr ""
 
 msgid "Privacy Crash Course"
-msgstr ""
+msgstr "Rapidkurso pri privateco"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3112]:
```suggestion
msgstr "Rapidkurso Pri Privateco"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 31]
```diff
@@ -3114,7 +3123,7 @@ msgid "Privacy Newsletter"
 msgstr "Privateca bulteno"
 
 msgid "Privacy Policy"
-msgstr ""
+msgstr "Privateca politiko"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3126]:
```suggestion
msgstr "Privateca Politiko"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 32]
```diff
@@ -3124,29 +3133,29 @@ msgid "Privacy Settings"
 msgstr "Privatecaj agordoj"
 
 msgid "Privacy for %s"
-msgstr ""
+msgstr "Privateco por %s"
 
 msgctxt "SERP footer content"
 msgid "Privacy in Your Inbox"
-msgstr ""
+msgstr "Privateco en via enirkesto"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3140]:
```suggestion
msgstr "Privateco en Via Enirkesto"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 33]
```diff
@@ -3124,29 +3133,29 @@ msgid "Privacy Settings"
...
 
 msgctxt "showcase_newsletter"
 msgid "Privacy in Your Inbox"
-msgstr ""
+msgstr "Privateco en via enirkesto"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3144]:
```suggestion
msgstr "Privateco en Via Enirkesto"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 34]
```diff
@@ -3124,29 +3133,29 @@ msgid "Privacy Settings"
...
 msgid "Privacy, simplified"
-msgstr ""
+msgstr "Privateco, simpligita"
 
 msgid "Privacy, simplified."
 msgstr "Privateco, simpligita."
 
 msgid "Private Search"
-msgstr ""
+msgstr "Privata serĉo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3155]:
```suggestion
msgstr "Privata Serĉo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 35]
```diff
@@ -3124,29 +3133,29 @@ msgid "Privacy Settings"
...
 
 msgid "Private Search Engine"
-msgstr ""
+msgstr "Privata serĉilo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3158]:
```suggestion
msgstr "Privata Serĉilo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 36]
```diff
@@ -3167,19 +3176,19 @@ msgstr "Invitu min anonime uzi retumilan pozicion por pli bonaj rezultoj."
...
+msgstr "Invitu min adapti mian DuckDuckGo-sperton."
 
 msgctxt "SERP footer content"
 msgid "Protect Your Devices"
 msgstr "Protektu viajn aparatojn"
 
 msgctxt "showcase_privacy"
 msgid "Protect Your Devices"
-msgstr "Protekti viajn aparatojn"
+msgstr "Protektu viajn aparatojn"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3191]:
```suggestion
msgstr "Protektu Viajn Aparatojn"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 37]
```diff
@@ -3277,16 +3286,16 @@ msgstr "Regiono:"
 
 msgctxt "maps_places"
 msgid "Regular Hours Today"
-msgstr ""
+msgstr "Kutimaj horoj hodiaŭ"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3289]:
```suggestion
msgstr "Kutimaj Horoj Hodiaŭ"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 38]
```diff
@@ -3277,16 +3286,16 @@ msgstr "Regiono:"
...
 
 msgid "Related Searches"
-msgstr ""
+msgstr "Rilataj serĉoj"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3292]:
```suggestion
msgstr "Rilataj Serĉoj"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 39]
```diff
@@ -3332,14 +3341,14 @@ msgstr "Marko-koloro de la rezultoj"
 
 msgctxt "settings"
 msgid "Result Hover, Module, and Dropdown Background Color"
-msgstr ""
+msgstr "Rezulta musumado, modulo kaj la fonkoloro de la falmenuo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3344]:
```suggestion
msgstr "Rezulta Musumado, Modulo kaj la Fonkoloro de la Falmenuo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 40]
```diff
@@ -3481,12 +3490,12 @@ msgid ""
...
 "%sAdd new%s)"
 msgstr ""
-"Rulumu malsupren kaj trovu na %sSerĉu en la adresa breto%s. Klaku sur %sŜanĝu"
+"Rulumu malsupren kaj trovu %sSerĉu en la adresa breto%s. Klaku sur %sŜanĝu"
 "%s (aŭ %sAldonu novan%s)"
 
 msgctxt "settings"
 msgid "Seach Visibility"
-msgstr ""
+msgstr "Serĉa videblo"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3498]:
```suggestion
msgstr "Serĉa Videblo"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 41]
```diff
@@ -3510,15 +3519,15 @@ msgstr "Serĉbutona fono"
 #. search for a place within the map area currently shown in expanded map
 msgctxt "vertical_map"
 msgid "Search Current Map Area"
-msgstr ""
+msgstr "Traserĉi la aktualan mapan areon"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3522]:
```suggestion
msgstr "Traserĉi la Aktualan Mapan Areon"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 42]
```diff
@@ -3510,15 +3519,15 @@ msgstr "Serĉbutona fono"
...
 
 msgid "Search DuckDuckGo"
 msgstr "DuckDuckGo serĉi"
 
 #. search for a place near the user's location
 msgctxt "vertical_map"
 msgid "Search Near My Location"
-msgstr ""
+msgstr "Serĉi proksime de mia loko"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3530]:
```suggestion
msgstr "Serĉi Proksime De Mia Loko"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 43]
```diff
@@ -3711,14 +3720,14 @@ msgstr "Agordi DuckDuckGo kiel vian %sdefaŭltan serĉilon"
 
 msgctxt "precise_user_location"
 msgid "Set Manually"
-msgstr ""
+msgstr "Agordi mane"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3723]:
```suggestion
msgstr "Agordi Mane"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 44]
```diff
@@ -3759,37 +3768,37 @@ msgid "Short"
...
+#. http://i.imgur.com/msbR8Xk.png
+#. Clicking will show you your Settings as a URL and JSON like this http://i.imgur.com/59sqEFU.png
 msgctxt "setting"
 msgid "Show Bookmarklet and Settings Data"
 msgstr "Montri legosigneton kaj agordajn datumojn"
 
 msgctxt "mobile expanded map"
 msgid "Show Detail"
-msgstr ""
+msgstr "Montri detalojn"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3781]:
```suggestion
msgstr "Montri Detalojn"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 45]
```diff
@@ -3759,37 +3768,37 @@ msgid "Short"
...
 
 msgctxt "expand_text"
 msgid "Show Less"
 msgstr "Montri malpli"
 
 msgctxt "mobile expanded map"
 msgid "Show List"
-msgstr ""
+msgstr "Montri liston"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3789]:
```suggestion
msgstr "Montri Liston"
```

File: locales/eo_XX/LC_MESSAGES/duckduckgo.po [hunk 46]
```diff
@@ -3759,37 +3768,37 @@ msgid "Short"
...
 
 msgctxt "expand_text"
 msgid "Show More"
 msgstr "Montri pli"
 
 msgctxt "load_more"
 msgid "Show More"
-msgstr "Montru Pli"
+msgstr "Montri pli"
```
Comment (GioSensation) [inline locales/eo_XX/LC_MESSAGES/duckduckgo.po:3797]:
```suggestion
msgstr "Montri Pli"
```

**Comment by GioSensation** [comment]
Thanks for your help with this, @RobinvanderVliet. In the future, please try to keep the capitalization of the original string. I've fixed it myself and I will go on and merge this. 🎉

**Comment by RobinvanderVliet** [comment]
@GioSensation

Thank you for merging it, but I have a comment on the capitalization style.

The capitalization style you applied to some sentences is very uncommon and foreign in Esperanto, I would recommend undoing it. We only capitalize the first word of sentences and proper nouns.

Is such capitalization neccesary? In my native language, Dutch (nl), we also don't capitalize words like that. The [Dutch DuckDuckGo translation](https://github.com/duckduckgo/duckduckgo-locales/blob/master/locales/nl_NL/LC_MESSAGES/duckduckgo.po) also does not capitalize words like that.

**Pull request merged**